src/megachatapi_impl.h

```diff
@@ -264,14 +272,15 @@ class MegaChatCallPrivate : public MegaChatCall
     MegaChatHandle callid;
     karere::AvFlags localAVFlags;
     karere::AvFlags initialAVFlags;
-    int changed;
+    int changed = MegaChatCall::CHANGE_TYPE_NO_CHANGES;
     int64_t initialTs;
     int64_t finalTs;
     std::string temporaryError;
     std::map<chatd::EndpointId, MegaChatSession *> sessions;
     std::map<chatd::EndpointId, karere::AvFlags> participants;
-    MegaChatHandle peerId;  // to identify the updated session
-    uint32_t clientid;  // to identify the updated session
+    MegaChatHandle peerId;  // to identify participant added or removed
+    uint32_t clientid;  // to identify the participant added or removed
+    bool addedOrRemoved = false;
```

Comment:
This kind of booleans should be avoided. They are not easy to understand. Based on the name of the variable, a developer should be able to guess the meaning. `if (addedOrRemoved)` doesn't result in a good readability

Reply:
Fixed